.githooks/pre-commit

```diff
@@ -1,5 +1,3 @@
 #!/bin/sh
 
-echo "We are now linting on every commit. You can skip this with the '--no-verify' flag."
-
-cd templates/vue && npm run lint -- --fix
\ No newline at end of file
+cd templates/vue && node lint.js
```

Comment:
Can we actually add a flag to allow user to skip linting when committing?

Reply:
Users can do this already with the `--no-verify` flag. See [git docs](https://git-scm.com/docs/githooks#_pre_commit).